DS13/sound/effects/section31/credits.txt

```diff
@@ -0,0 +1,2 @@
+Credits for the edited clip to Revkor:
```

Comment:
Is this the appropriate place for credits?

Reply:
I don't care.